components/webhooks/pkg/server/activation.go

```diff
@@ -15,49 +16,44 @@ import (
 func (h *serverHandler) activateOneConfigHandle(w http.ResponseWriter, r *http.Request) {
 	id := chi.URLParam(r, PathParamId)
 	c, err := h.store.UpdateOneConfigActivation(r.Context(), id, true)
-	if err == nil {
-		logging.FromContext(r.Context()).Infof("PUT %s/%s%s", PathConfigs, id, PathActivate)
+	if err == nil || errors.Is(err, storage.ErrConfigNotModified) {
```

Comment:
When the h.store.UpdateOneConfigActivation function is returning an error, the config `c` is set to empty, and will be then send as a response, is it what we want ?